src/components/Footer/SocialButton/index.tsx

```diff
@@ -11,7 +11,7 @@ interface SocialButtonProps {
 
 const SocialButton: React.FC<SocialButtonProps> = ({ Icon, href, label }) => (
   <Link href={href} passHref>
-    <Button aria-label={label}>
+    <Button aria-label={label} target="_blank">
```

Comment:
Prevent security issues and tabnabbing
For more info about: https://medium.com/sedeo/how-to-fix-target-blank-a-security-and-performance-issue-in-web-pages-2118eba1ce2f
```suggestion
    <Button aria-label={label} target="_blank" rel=”noopener noreferrer”>
```